src/main/api/2/studio-api-2.yaml

```diff
@@ -1027,6 +1027,88 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'
 
+  /users/me/sites/{site}/permissions:
+    get:
+      tags:
+        - users
+      summary: Get the site permissions for the current authenticated user
+      description: 'Required Permission: "LOGGED_IN"'
+      operationId: getCurrentUserSitePermissions
+      parameters:
+        - name: site
+          in: path
+          description: The site ID
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  response:
+                    $ref: '#/components/schemas/ApiResponse'
+                  'permissions':
+                    type: array
+                    items:
+                      type: string
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+
+  /users/me/sites/{site}/hasPermissions:
+    get:
+      tags:
+        - users
+      summary: Check if the current authenticated user has given permissions
+      description: 'Required Permission: "LOGGED_IN"'
+      operationId: hasCurrentUserSitePermssions
+      parameters:
+        - name: site
```

Comment:
We had siteId in other APIs. Change to `siteId` for consistency.

Reply:
all users APIs have same naming (`site` not `siteId`)